cristin-project/src/main/java/no/unit/nva/cristin/projects/model/nva/NvaProject.java

```diff
@@ -238,22 +239,22 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "NvaProject{" +
-                "context='" + context + '\'' +
-                ", id=" + id +
-                ", type='" + type + '\'' +
-                ", identifiers=" + identifiers +
-                ", title='" + title + '\'' +
-                ", language=" + language +
-                ", alternativeTitles=" + alternativeTitles +
-                ", startDate=" + startDate +
-                ", endDate=" + endDate +
-                ", funding=" + funding +
-                ", coordinatingInstitution=" + coordinatingInstitution +
-                ", contributors=" + contributors +
-                ", status=" + status +
-                ", academicSummary=" + academicSummary +
-                ", popularScientificSummary=" + popularScientificSummary +
-                '}';
+        return "NvaProject{"
+                + "context='" + context + '\''
```

Comment:
Manual Json? we have JsonSerializable. you can also add a type if you want to know which type the object is.

Reply:
Actualli it was a IntelliJ generated toString (for debugging) , but JsonSerializable is way better